src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1095,8 +1138,18 @@ async def verify_identification_code(
     )
     try:
         privacy_request.verify_identity(db, provided_code.code)
+        policy: Optional[Policy] = Policy.get(
+            db=db, object_id=privacy_request.policy_id
+        )
+        if config.notifications.send_request_receipt_notification:
+            _send_privacy_request_receipt_email_to_user(
+                db, policy, privacy_request.get_persisted_identity().email
+            )
```

Comment:
Thanks for adding this case here, so the user gets the request receipt email when they verify their identity